spec/javascripts/report-a-problem-form-spec.js

```diff
@@ -30,9 +30,13 @@ describe("form submission for reporting a problem", function () {
     it("should submit using ajax to the action specified by the form", function() {
       var args;
       spyOn($, "ajax");
+      spyOn(GOVUK.analytics, "trackEvent");
       $form.triggerHandler('submit');
 
       expect($.ajax).toHaveBeenCalled();
+      expect(GOVUK.analytics.trackEvent).toHaveBeenCalledWith(
```

Comment:
The reason why we didn't do that is because `GOVUK.analytics.trackEvent` is already tested in `analytics/static-analytics-spec.js`.